src/notation/view/percussionpanel/percussionpanelmodel.cpp

```diff
@@ -125,28 +128,24 @@ void PercussionPanelModel::init()
 
 QList<QVariantMap> PercussionPanelModel::layoutMenuItems() const
 {
-    const TranslatableString padNamesTitle("notation/percussion", "Pad names");
-    // Using IconCode for this instead of "checked" because we want the tick to display on the left
-    const int padNamesIcon = static_cast<int>(m_useNotationPreview ? IconCode::Code::NONE : IconCode::Code::TICK_RIGHT_ANGLE);
+    static const auto padNamesTitle = TranslatableString("notation/percussion", "Pad names");
 
-    const TranslatableString notationPreviewTitle("notation/percussion", "Notation preview");
-    // Using IconCode for this instead of "checked" because we want the tick to display on the left
-    const int notationPreviewIcon = static_cast<int>(m_useNotationPreview ? IconCode::Code::TICK_RIGHT_ANGLE : IconCode::Code::NONE);
+    static const auto notationPreviewTitle = TranslatableString("notation/percussion", "Notation preview");
 
-    const TranslatableString editLayoutTitle = m_currentPanelMode == PanelMode::Mode::EDIT_LAYOUT
-                                               ? TranslatableString("notation/percussion", "Finish editing")
-                                               : TranslatableString("notation/percussion", "Edit layout");
-    const int editLayoutIcon = static_cast<int>(IconCode::Code::CONFIGURE);
+    static const auto editLayoutTitle = m_currentPanelMode == PanelMode::Mode::EDIT_LAYOUT
```

Comment:
This `static` depends on the non-static `m_currentPanelMode`

Reply:
Actually, in this method, there is no need to use `TranslatableString`, because the only thing you do with it, is calling `.qTranslated()`. So it should be possible to use just `qtrc`.